core/modules/views/tests/handlers/views_handler_field_bulk_form.test

```diff
@@ -0,0 +1,247 @@
+<?php
+
+/**
+ * @file
+ * Definition of ViewsHandlerFieldBulkFormTest.
+ */
+
+require_once BACKDROP_ROOT . '/core/modules/views/tests/views_query.test';
+
+class ViewsHandlerFieldBulkFormTest extends ViewsSqlTest {
+
+  public $view;
+
+  function setUp() {
+    parent::setUp();
```

Comment:
This causes `HP Fatal error:  Declaration of ViewsHandlerFieldBulkFormTest::setUp() must be compatible with ViewsSqlTest::setUp(array $modules = [])` (silent / hidden) problem.

Should probably rather be: `setUp($modules)`.